src/boot/boot.c

```diff
@@ -2974,6 +2974,19 @@ static EFI_STATUS discover_root_dir(EFI_LOADED_IMAGE_PROTOCOL *loaded_image, EFI
                 return open_volume(loaded_image->DeviceHandle, ret_dir);
 }
 
+static void set_log_level_from_smbios(void) {
+        if (is_confidential_vm())
+                return; /* Don't consume SMBIOS in CoCo contexts */
+
+        const char *level_str = smbios_find_oem_string("io.systemd.boot.loglevel=", /* after= */ NULL);
```

Comment:
please document this in the man pages, i.e. systemd-boot(7) and smbios-type-11(7)